docs/how-to-setup-wsl.md

```diff
@@ -0,0 +1,218 @@
+# Setting up WSL for Working with freeCodeCamp Locally
+
+Follow these guidlines for setting up WSL to set up freeCodeCamp locally.
```

Comment:
It would be a nice to add some requirements of WSL itself. I might be mistaken, but it requires you to be on Windows 10 Pro and I think WSL2 needs a may 2020 update.

Reply:
I would be fine doing this, but I left it out, seeing as Microsoft take users through the requirements on the WSL Docs page. Should I just add a small note anyway?

Reply:
Yeah, that would be useful because users can avoid going through all the caveats that Microsoft has with the 'Pro' version of OS and the need for being on the latest update.